pkgs/desktops/gnome/extensions/collisions.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
I am committing this because the diff is really useful when doing updates. I thought about adding a test that checks for collision handling, but I personally don't need it for maintaining.